ui/src/shared/reducers/currentDashboard.ts

```diff
@@ -0,0 +1,24 @@
+import {Action, SET_DASHBOARD} from 'src/shared/actions/currentDashboard'
+
+export interface CurrentDashboardState {
+  dashboard: string
```

Comment:
```suggestion
id: string
```
`state.currentDashboard.dashboard` reads a bit oddly.  Also, I don't know if its the `id`, `name`, or the whole `dashboard`.  Also `state.currentDashboard.id` looks and sounds exactly like what it is.

Reply:
changed it to `id`. totally makes more sense